tests/wpunit/Common/Asset_Test.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+namespace TEC\Common;
+
+use Codeception\TestCase\WPTestCase;
+use TEC\Common\StellarWP\Assets\Asset as Stellar_Asset;
+use TEC\Common\StellarWP\Assets\Assets;
+use TEC\Common\StellarWP\Assets\Config;
+use Tribe\Tests\Traits\With_Uopz;
+
+/**
+ * Class Opt_InTest
+ *
+ * @since 5.1.13
+ *
+ * @package TEC\Common\Telemetry
+ */
+class Asset_Test extends WPTestCase {
+	use With_Uopz;
+
+	protected $request_args = [];
+
+	protected static array $back_up = [];
+
+	/**
+	 * @before
+	 */
+	public function backup_and_set_up() {
+		self::$back_up = [
+			'hook_prefix' => Config::get_hook_prefix(),
+			'version'     => Config::get_version(),
+			'path'        => Config::get_path(),
+			'relative'    => Config::get_relative_asset_path(),
+		];
+
+		Config::set_hook_prefix( 'bork' );
+		Config::set_version( '1.0.0' );
+		Config::set_path( dirname( __DIR__, 3 ) );
```

Comment:
Use `codecept_root_dir` here? If I understand this correctly, this is to get to Common root.